### PR TITLE
[WMCO-3.x]: Revealed the hidden WMCO 3.x content for 4.8+

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR reverses #34783 and reveals all of the hidden WMCO content.

Version: 4.8+

Preview: https://deploy-preview-35137--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-2-x.html

### CODE REVIEW

File: windows_containers/windows-containers-release-notes-2-x.adoc [hunk 1]
```diff
@@ -61,8 +61,7 @@ This release adds the following new features and improvements.
 
 [id="wmco-2-0-0-vsphere-support"]
 ==== Support for clusters running on VMware vSphere
-
-You can now run Windows nodes on a cluster that is installed on VMware vSphere version 6.5, 6.7, or 7.0. You can create a Windows MachineSet object on vSphere to host Windows Server compute nodes.
+You can now run Windows nodes on a cluster installed on VMware vSphere version 6.5, 6.7, or 7.0. You can create a Windows MachineSet object on vSphere to host Windows Server compute nodes. For more information, see xref:../windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.adoc#creating-windows-machineset-vsphere[Creating a Windows `MachineSet` object on vSphere].
```
Comment (jeana-redhat) [inline windows_containers/windows-containers-release-notes-2-x.adoc:64]:
```suggestion
You can now run Windows nodes on a cluster installed on VMware vSphere version 6.5, 6.7, or 7.0. You can create a Windows `MachineSet` object on vSphere to host Windows Server compute nodes. For more information, see xref:../windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.adoc#creating-windows-machineset-vsphere[Creating a Windows `MachineSet` object on vSphere].
```

File: windows_containers/disabling-windows-container-workloads.adoc [hunk 2]
```diff
@@ -15,3 +15,4 @@ include::modules/deleting-wmco-namespace.adoc[leveloffset=+1]
 == Additional Resources
 
 * xref:../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[Deleting Operators from a cluster]
+* xref:../windows_containers/removing-windows-nodes.adoc#removing-windows-nodes[Removing Windows nodes].
```
Comment (jeana-redhat) [inline windows_containers/disabling-windows-container-workloads.adoc:18]:
```suggestion
* xref:../windows_containers/removing-windows-nodes.adoc#removing-windows-nodes[Removing Windows nodes]
```

File: windows_containers/windows-containers-release-notes-2-x.adoc [hunk 3]
```diff
@@ -76,4 +75,4 @@ Windows nodes are now fully integrated with most of the monitoring capabilities
 
 * The Prometheus windows_exporter used by the WMCO currently collects metrics through HTTP, so it is considered unsafe. You must ensure that only trusted users can retrieve metrics from the endpoint. The windows_exporter feature recently added support for HTTPS configuration, but this configuration has not been implemented for WMCO. Support for HTTPS configuration in the WMCO will be added in a future release.
 
-* The kube-proxy service terminates unexpectedly after the load balancer is created if you create the load balancer after the Windows pods begin running. To avoid this issue, you must create the load balancer before the Windows pods statuses transition to `Running`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939968[*BZ#1939968*])
+* The kube-proxy service terminates unexpectedly after the load balancer is created if you create the load balancer after the Windows pods begin running. To avoid this issue, you must create the load balancer before the Windows pods status transitions to `Running`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939968[*BZ#1939968*])
```
Comment (jeana-redhat) [inline windows_containers/windows-containers-release-notes-2-x.adoc:78]: > the Windows pods status transitions

Confused about subject-verb in the change here - wouldn't pods have statuses not status?

**Review by jeana-redhat — COMMENTED** [review]
A few small things, looking good though

File: installing/installing_aws/installing-aws-network-customizations.adoc [hunk 4]
```diff
@@ -68,7 +68,7 @@ include::modules/configuring-hybrid-ovnkubernetes.adoc[leveloffset=+1]
 
 [NOTE]
 ====
-For more information on using Linux and Windows nodes in the same cluster, see Understanding Windows container workloads.
+For more information on using Linux and Windows nodes in the same cluster, see xref:../../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[Understanding Windows container workloads].
```
Comment (aravindhp) [inline installing/installing_aws/installing-aws-network-customizations.adoc:71]: Don't we also need this note for https://github.com/openshift/openshift-docs/blob/main/installing/installing_vsphere/installing-vsphere-network-customizations.adoc?

**Pull request merged**